pages/main.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <meta charset="utf-8">
+  <title>JIRA Quick Search</title>
+  <meta name="description" content="">
+  <meta name="author" content="">
+
+  <!-- Mobile Specific Metas -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href="http://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="../css/normalize.css">
+  <link rel="stylesheet" href="../css/skeleton.css">
+  <link rel="stylesheet" href="../css/styles.css">
+  <link rel="icon" type="image/png" href="../images/favicon.png">
+
+  <script src="../js/alerts.js"></script>
+  <script src="../js/fields.js"></script>
+  <script src="../js/tabs.js"></script>
+  <script src="../js/functions.js"></script>
+  <script src="../js/http.js"></script>
+  
+  <script src="../scripts/result-bindings.js"></script>
+  <script src="../scripts/main.js"></script>
+
+</head>
+<body>
+  <div class="container">
+
+    <!-- LOADING -->
+    <div id="loading-container" hidden>Loading...</div>
+
+    <!-- LOGO -->
+    <img src="../images/logo.png" class="logo-image">
+
+    <!-- ALERTS -->
+    <div id="alert-container"></div>
+
+    <!-- TABS -->
+    <div class="tab">
+      <button class="tablinks" id="tab-status">Ticket Status</button>
+      <button class="tablinks" id="tab-activity">Activity Query</button>
+    </div>
+    
+    <!-- TAB -->
+    <div id="tab-status" class="tabcontent open">
+        <label>Project name <span class="required">*</span></label>
```

Comment:
What is the reasoning behind requiring the project name? What if the user wants to query all projects that have in progress tickets for example?